jack.html

```diff
@@ -0,0 +1,252 @@
+                        </div><!--//level-bar-->                                 
+                    </div><!--//item-->
+
+                    <div class="item">
+                        <h3 class="level-title">SQL Server (TSQL)</h3>
+                        <div class="level-bar">
+                            <div class="level-bar-inner" data-level="80%">
+                            </div>                                      
+                        </div><!--//level-bar-->                                 
+                    </div><!--//item-->
+                    
+                    <div class="item">
+                        <h3 class="level-title">ASP.NET</h3>
+                        <div class="level-bar">
+                            <div class="level-bar-inner" data-level="60%">
+                            </div>                                      
+                        </div><!--//level-bar-->                                 
+                    </div><!--//item-->
+                    
+                    <div class="item">
+                        <h3 class="level-title">Agile / Scrum</h3>
+                        <div class="level-bar">
+                            <div class="level-bar-inner" data-level="90%">
+                            </div>                                      
+                        </div><!--//level-bar-->                                 
+                    </div><!--//item-->
+                    
+				    <div class="item">
+                        <h3 class="level-title">Unit Testing</h3>
+                        <div class="level-bar">
+                            <div class="level-bar-inner" data-level="80%">
+                            </div>                                      
+                        </div><!--//level-bar-->                                 
+                    </div><!--//item-->                                
+                    
+                </div>  
+            </section><!--//skills-section-->
+            
+            <section class="section experiences-section">
+                <h2 class="section-title"><i class="fa fa-briefcase"></i>Ervaring</h2>
+
+                <div class="item">
+                    <div class="meta">
+                        <div class="upper-row">
+                            <h3 class="job-title">Senior Developer</h3>
+                            <div class="time"><i class="fa fa-calendar"></i> 2017 - 2018</div>
+                        </div><!--//upper-row-->
+                        <div class="company">Raet B.V.</div>
+                    </div><!--//meta-->
+                    <div class="details">
+                        <p>In dienst bij deze werkgever heeft Jack in teamverband on RESTful API ontwikkeld voor een zeer grote klant. Tijdige oplevering, stabiliteit en performance van deze API zijn hierbij de belangrijkste speerpunten geweest.
```

Comment:
- "in teamverband **on** RESTful API"
- "internationalem"
- "Waar nodig heeft zaken communiceerd"
- "Vanwege de go/no-go project'-aard " >> opening quote ontbreekt
- "heeft zich hard gemaakt om de API" >> heeft hij zich?
- "gegemaakt"